terraform/rancher/project/postgresql.tf

```diff
@@ -53,9 +53,9 @@ resource "rancher2_app_v2" "postgresql" {
       replicaCount: 1
       resources:
         requests:
-          memory: 512Mi
+          memory: 2048Mi
         limits:
-          memory: 10240Mi
```

Comment:
in one of our commits, there was a variable implementation for specifying desired amount of RAM for PSQL STS, why do we skip it?

Reply:
If I am not mistake, implemented ROM variable for storage, not for RAM